### PR TITLE
Create PAPI placeholders for spawners

### PR DESCRIPTION
**Pull request by rglautaro**
Hello! I'm using this plugin and find it great for my server, specially the performance improvements by allowing players to setup virtual mob farms instead of stacking and killing hundred of mobs.
I wanted to make some custom items that would require to know some info about the amount and limit but noticed you don't have any placeholders yet. I hope this can be added!

`%spawnercollectors_amount_<entity-type>%`  for the amount of spawners the player has of a given type
`%spawnercollectors_amount%`  for the total amount of spawners the player has of all types

`%spawnercollectors_limit%`  for the max spawners a player may have as configured in the config.yml
`%spawnercollectors_limit_<entity-type>%`  for the max spawners of a given type a player may have as configured using permissions

### CODE REVIEW

File: src/main/java/me/bestem0r/spawnercollectors/placeholders/PlaceholderAPIExpansion.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    @Override
+    public String getIdentifier() {
+        return "spawnercollectors";
+    }
+
+    @Override
+    public String getAuthor() {
+        return "bestem0r";
```
Comment (Bestem0r) [inline src/main/java/me/bestem0r/spawnercollectors/placeholders/PlaceholderAPIExpansion.java]: Can you change this to capital `B`? 😊

**Comment by Bestem0r** [comment]
Looks very good! Thanks

File: src/main/java/me/bestem0r/spawnercollectors/placeholders/PlaceholderAPIExpansion.java (hunk 1 above)
Comment (rglautaro) [inline src/main/java/me/bestem0r/spawnercollectors/placeholders/PlaceholderAPIExpansion.java]:
```suggestion
        return "Bestem0r";
```

Comment (rglautaro) [reply src/main/java/me/bestem0r/spawnercollectors/placeholders/PlaceholderAPIExpansion.java]: No problem!

**Pull request merged**